hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -614,6 +615,57 @@ def hpat_pandas_series_sub_number_impl(self, other):
     raise TypingError('{} The object must be a pandas.series or scalar. Given other: {}'.format(_func_name, other))
 
 
+@overload_method(SeriesType, 'sum')
+def hpat_pandas_series_sum(self, axis=None, skipna=None, level=None, numeric_only=None, min_count=0, **kwargs):
+    """
+    Pandas Series method :meth:`pandas.Series.sum` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sum1
+    #    Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sum2
+
+    Parameters
+    ----------
+    self: :class:`pandas.Series`
+        input arg
+    axis:
+        *unsupported*
+    skipna:
```

Comment:
it looks like you can support `skipna` because you can use Numpy functions for each parameter value